build_web_compilers/lib/src/dev_compiler_bootstrap.dart

```diff
@@ -62,9 +62,13 @@ https://github.com/dart-lang/build/blob/master/docs/faq.md#how-can-i-resolve-ski
   // See https://github.com/dart-lang/sdk/issues/27262 for the root issue
   // which will allow us to not rely on the naming schemes that dartdevc uses
   // internally, but instead specify our own.
-  var appModuleScope = toJSIdentifier(
+  var simpleAppModuleScope = toJSIdentifier(
```

Comment:
nit: rename to `oldAppModuleScope` to match the param and a bit more accurately describe it